src/torque-offset-estimator.cpp

```diff
@@ -81,22 +80,22 @@ namespace dynamicgraph
         // accSignals.clear();
         // gyrSignals.clear();
         // tauSignals.clear();
-        stdVecJointTorqueOffsets.clear();
+        // stdVecJointTorqueOffsets.clear();
       }
 
 
       /* --- COMMANDS ---------------------------------------------------------- */
       /* --- COMMANDS ---------------------------------------------------------- */
       /* --- COMMANDS ---------------------------------------------------------- */
       void TorqueOffsetEstimator::init(const std::string &urdfFile,
-                                       const Eigen::Matrix4d& _m_torso_X_imu)
+                                       const Eigen::Matrix4d& m_torso_X_imu_)
       {
         try {
           se3::urdf::buildModel(urdfFile,se3::JointModelFreeFlyer(),m_model);
           // assert(m_model.nq == N_JOINTS+7);
           // assert(m_model.nv == N_JOINTS+6);
 
-          jointTorqueOffsets.resize(m_model.nv-6);
+          jointTorqueOffsets.resize(m_model.nv);
```

Comment:
Why did you change the size of the torque offset? I don't see the reason of computing it for the base axes.